docs/ensembl-help/using-ensembl/getting-ensembl-data/toc.yml

```diff
@@ -0,0 +1,2 @@
+- name: Tools and APIs
```

Comment:
Why did we call this "Tools and apis"? Were there some instructions from Andrea, Outreach, or @ens-st3 about this?

_(Compare with BLAST, which is a tool,  but is in a different section)_

Reply:
Andrea mentioned this [here](https://genomes-ebi.slack.com/archives/C010X2BA56U/p1658403863809939).